docs-src/cloud-context/saml-how-to-configure-with-azure-ad.md

```diff
@@ -0,0 +1,54 @@
+---
+id: saml-how-to-configure-with-azure-ad
+title: How to configure SAML with Azure AD
+sidebar_label: Configure SAML with Azure AD
+description: To use Azure AD as your SAML IdP, create an Azure AD Enterprise application.
+tags:
+  - guide-context
+---
+
+To use Azure AD as your SAML IdP, create an Azure AD Enterprise application.
+
+1. Sign in to the [Microsoft Azure AD portal](https://portal.azure.com/).
+1. On the home page under **Manage Azure Active Directory**, select **View**.
+1. On the **Overview** page near the top, select **Add > Enterprise application**.
+1. On the **Browse Azure AD Gallery** page near the top, select **Create your own application**.
+1. In the **Create your own application** pane, provide a name for your application (such as `temporal-cloud`) and select **Integrate any other application you don't find in the gallery**.
+1. Select **Save**.
+1. In the **Getting Started** section, select **2. Set up single sign on**.
+1. On the **Single sign-on** page, select **SAML**.
+1. In the **Basic SAML Configuration** section of the **SAML-based Sign-on** page, select **Edit**.
+1. In **Identifier (Entity ID)**, enter the following entity identifier, including your Account Id where indicated:
+
+   ```bash
+   urn:auth0:prod-tmprl:ACCOUNT_ID-saml
+   ```
+
+   A correctly formed entity identifier resembles the following:
+
+   ```bash
+   urn:auth0:prod-tmprl:f45a2-saml
```

Comment:
idk how i feel about this the odds of us generating the same id again is 1/60 million...

Reply:
We've used this Account Id elsewhere in the docs, so if we need to change it to something explicitly reserved, I can quickly locate and update all instances.